lang/cs/Org.Apache.REEF.IO/PartitionedData/FileSystem/FileSystemInputPartition.cs

```diff
@@ -71,13 +70,22 @@ public string Id
         /// <summary>
         /// Caches from the remote File System to a local disk.
         /// </summary>
-        public void Cache()
+        public CacheLevel Cache(CacheLevel level)
```

Comment:
I didn't want to make the change too big to review. I'd like to defer this to another PR. What do you think?